frameworks/kafka/client/Dockerfile

```diff
@@ -6,14 +6,14 @@ RUN apt-get update && \
 
 ARG JAVA_TAR_ARCHIVE="jre-8u152-linux-x64-jce-unlimited.tar.gz"
```

Comment:
Probably can do this in a different PR but since we updated the JRE to 172 for all packages in #2488 we may want to keep the testing client in step

Reply:
I didn't want to change the clients in this PR, just add them to the repo. We can follow up with an update PR.

Reply:
sounds good... and sounds like we'll be moving this to a separate repo anyhow